tests/phpunit/tests/option/networkOption.php

```diff
@@ -794,60 +794,37 @@ public function test_update_network_option_should_conditionally_apply_site_and_o
 	}
 
 	/**
-	 * Tests that update_network_option() adds a non-existent option that uses a filtered default value.
+	 * Tests that update_network_option() does not add a non-existent option that uses a filtered default value in multisite.
 	 *
 	 * @ticket 59360
```

Comment:
The `ms-required` group is skipped so this will reduce noise in the singe site test runs

https://github.com/WordPress/wordpress-develop/blob/9cec1601108c5b7e5b76aaecd88eb4b3d4680e1e/phpunit.xml.dist#L29

```suggestion
	 * @ticket 59360
	 *
	 * @group ms-required
```

Reply:
Good call! We should do this in the rest of the tests, though I'll leave adjusting the other tests out of this PR to avoid going beyond its scope.

Reply:
Updated in [49748a5](https://github.com/WordPress/wordpress-develop/pull/5465/commits/49748a5439d5ad18d52abe087ebb6c56a6fca565). Also removed the condition block as the test isn't executed at all when run with the Single Site configuration.